content/security/rate-limiting.md

```diff
@@ -166,7 +166,7 @@ export class GqlThrottlerGuard extends ThrottlerGuard {
   getRequestResponse(context: ExecutionContext) {
     const gqlCtx = GqlExecutionContext.create(context);
     const ctx = gqlCtx.getContext();
-    return { req: ctx.req, res: ctx.res };
+    return { req: ctx.req, res: ctx.req.res };
```

Comment:
Instead of accessing `res` through the `req` object, `res` should be added to the GraphQL context via `context: ({ req, res }) => ({ req ,res })` in the `GraphQLModule`'s context option

Reply:
Is that mentioned somewhere in the docs?
I simply followed the docs, and it led to the mentioned error.